src/components/CallView/shared/VideoBackground.vue

```diff
@@ -145,7 +185,32 @@ export default {
 	methods: {
 		// Calculate the background blur based on the height of the background element
 		setBlur({ width, height }) {
-			this.blur = this.$store.getters.getBlurFilter(width, height)
+			this.blur = this.$store.getters.getBlurRadius(width, height)
+		},
+
+		generateBlurredBackgroundImage() {
```

Comment:
Or just a plain background?

Reply:
I think that if we can use a blurred background we should use it, but of course if the performance hit is still bad even with an OffscreenCanvas then the plain background could be the only solution.